tracer/tools/PipelineMonitor/PipelineMonitor.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <OutputType>Exe</OutputType>
+        <TargetFramework>net6.0</TargetFramework>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <PackageReference Include="Datadog.Trace" Version="2.6.0" />
```

Comment:
We should probably add this to the "version updates" script if we're going to use the NuGet package

Reply:
If there's isn't a strong need to use the NuGet package, this could instead be a ProjectReference to the `Datadog.Trace.csproj`